Package.swift

```diff
@@ -54,5 +54,6 @@ let package = Package(
         "StorybookKit",
       ]
     ),
-  ]
+  ],
+  swiftLanguageVersions: [.v5]
```

Comment:
we can't add also v6?

Reply:
Not if we don't fix a bunch of other stuff